lib/meadow/utils/data_loader.ex

```diff
@@ -13,6 +13,8 @@ defmodule Meadow.Utils.DataLoader do
     WorkDescriptiveMetadata
```

Comment:
```suggestion
```

Small request, you removed the reference to `WorkDescriptiveMetadata` so you can yank it from the alias list.